modules/common/dialect.py

```
@@ -0,0 +1,154 @@
+            return la if la is not None else ra
+
+        def _humidity():
+            lh = left.get("hu") if not _is_sentinel_float(left.get("hu")) else None
+            rh = right.get("hu") if not _is_sentinel_float(right.get("hu")) else None
+            return lh if lh is not None else rh
+
+        return {
+            "ts": ts,
+            "ambient_temp": _to_centidegrees(_ambient()),
+            "mcu_temp": _to_centidegrees(rec.get("mcu")),
+            "humidity": _to_centidegrees(_humidity()),  # both percent and degrees use ×100
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)